src/SupabaseStorageClient.ts

```diff
@@ -15,3 +15,8 @@ export class SupabaseStorageClient extends StorageBucketApi {
     return new StorageFileApi(this.url, this.headers, id, this.fetch)
   }
 }
+
+//! Alias `StorageClient` as `SupabaseStorageClient` for backwards compatibility.
+export {
+  StorageClient as SupabaseStorageClient,
+}
```

Comment:
Could you run `npm run format`? It should reformat this to `export { StorageClient as SupabaseStorageClient }`.

Reply:
Running that command made some changes in other files as well. Should i commit those too?